cypress/component/UserAllRecipes.cy.jsx

```diff
@@ -0,0 +1,117 @@
+import user1 from "../../src/assets/user/user1.svg";
+
+import user2 from "../../src/assets/user/user2.svg";
+import Pizza from "../../src/assets/pizza.jpg";
+import UserAllRecipes from "../../src/pages/profile/ProfileSections/UserAllRecipes";
+
+describe("UserAllRecipes", () => {
+  const user = {
```

Comment:
I think you should create a file with test data. Seems like you are duplicating this user among all your test files.